### DISCUSSION TITLE
is there a way to use DSL AC68U as AImesh node or repater with wired backhaul connection?

### DISCUSSION THREAD

**Discussion by junkkari3** [gnuton/asuswrt-merlin.ng#626] [Q&A]
I have two DSL AC68U which I would like to add as wired AI mesh nodes or repeaters. Is it possible? When they are in router mode, I can configure so that instead of the DSL port, it uses one of the four "ethernet out" (lan or what ever) ports as "ethernet in" port (wan or what ever). However, I do not want to use them in router mode, I want to add them to my existing wifi set up (router and mesh node) as mesh nodes, or as repeaters - with wired backhaul connection. Is this somehow possible with the latest gnuton merlin firmware, or with the (old) experimental repeater firmware? I.e. configuring the device in AI node / repeater mode to use wired backhaul (because of the DSL port this is not possible in the ASUS standard software).  Thank you anyway, for a long while I used these two devices with your built through the wireless connection. The build was great (but the devices started acting up so I bought new ones, and now want to add these as wired nodes)

**Comment by gnuton** [comment]
Hi.
The repeater mode is supported by one of the old experimental firmwares. Repeater mode is not as fast as aimesh although.
Aimesh via wan is not yet available on dsl-ac68u. This was requested some time back but I didn't have time to look at it. I guess that's maybe possible, bit I would need to try this out.

**Reply by junkkari3** [reply]
OK, thank you for the reply. It would be great if you ever have time to look into that!

**Reply by gnuton** [reply]
Please please open a ticket I will try to check this hopefully this weekend and see how much work there is. I think this can be achieved also by setting some nvram settings. But let me check better and I will come back with an estimate for the task and of it's feasible

**Reply by dlinkdns** [reply]
Please Gnuton I have a DSL-AC68U and have installed "DSL-AC68U__gnuton-snapshot-feature-repeater_DSL_1.0.5.3.trx". I am trying to get AiMesh to work from LAN wired network. setting up AiMEsh via wifi is ok but when i want to install my router at a futher distance and use LAN to connect from my current AX88U to DSL-AC68U i create a loop back. can you help
